prod/westeurope/eucovidcert/functions_eucovidcert/function_app/terragrunt.hcl

```diff
@@ -0,0 +1,89 @@
+dependency "resource_group" {
+  config_path = "../../resource_group"
+}
+
+dependency "subnet" {
+  config_path = "../subnet"
+}
+
+# Common
+dependency "application_insights" {
+  config_path = "../../../common/application_insights"
+}
+
+dependency "key_vault" {
+  config_path = "../../../common/key_vault"
+}
+
+
+# Include all settings from the root terragrunt.hcl file
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::git@github.com:pagopa/io-infrastructure-modules-new.git//azurerm_function_app?ref=v3.0.3"
+}
+
+locals {
+  commonvars                   = read_terragrunt_config(find_in_parent_folders("commonvars.hcl"))
+  app_insights_ips_west_europe = local.commonvars.locals.app_insights_ips_west_europe
+}
+
+inputs = {
+  name                = "eucovidcert"
+  resource_group_name = dependency.resource_group.outputs.resource_name
+
+  resources_prefix = {
+    function_app     = "fn3"
+    app_service_plan = "fn3"
+    storage_account  = "fn3"
+  }
+
+  app_service_plan_info = {
+    kind     = "elastic"
+    sku_tier = "ElasticPremium"
+    sku_size = "EP1"
+  }
+
+  runtime_version = "~3"
+
+  application_insights_instrumentation_key = dependency.application_insights.outputs.instrumentation_key
+
+  app_settings = {
+    FUNCTIONS_WORKER_RUNTIME       = "node"
+    WEBSITE_NODE_DEFAULT_VERSION   = "12.18.0"
+    WEBSITE_RUN_FROM_PACKAGE       = "1"
+    NODE_ENV                       = "production"
+
+    // Keepalive fields are all optionals
+    FETCH_KEEPALIVE_ENABLED             = "true"
+    FETCH_KEEPALIVE_SOCKET_ACTIVE_TTL   = "110000"
+    FETCH_KEEPALIVE_MAX_SOCKETS         = "40"
+    FETCH_KEEPALIVE_MAX_FREE_SOCKETS    = "10"
+    FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
+    FETCH_KEEPALIVE_TIMEOUT             = "60000"
+
+    APPINSIGHTS_SAMPLING_PERCENTAGE = 5
+
+    # this app settings is required to solve the issue:
+    # https://github.com/terraform-providers/terraform-provider-azurerm/issues/10499
+    WEBSITE_CONTENTSHARE = "io-p-fn3-eucovidcert-content"
+  }
+
+  app_settings_secrets = {
+    key_vault_id = dependency.key_vault.outputs.id
+    map = {
+    }
+  }
+
+  allowed_subnets = [
+    dependency.subnet.outputs.id,
```

Comment:
which resource is calling this function? Could it be the api management? If so we need to whitelist the apim subnet too.

Reply:
whitelist appbackendl1, appbackendl2

Reply:
We need to call _fn-services_ for sure, I don't know about the api management. @balanza can you help us?

Reply:
Added appbackendl1 and appbackendl2 [33ca10c](https://github.com/pagopa/io-infrastructure-live-new/pull/529/commits/33ca10ce7b6c70fb338f34d180f670d1c3d7a30f)

Reply:
@gquadrati, as @uolter suggest, apim will call this function so we need to whitelist also apim subnet

Reply:
Yes, sorry I forgot it. I created a new PR #531